packages/ove-app-svg/src/client/control/svg.js

```diff
@@ -0,0 +1,38 @@
+initControl = function (data) {
+    window.ove.context.isInitialized = false;
+    log.debug('Application is initialized:', window.ove.context.isInitialized);
+    log.debug('Restoring state:', data);
+    window.ove.state.current = data;
+    let url = OVE.Utils.getQueryParam('url');
+    if (url) {
+        log.debug('New URL at controller:', url);
+        // If a URL was passed, the URL of the loaded state would be overridden.
```

Comment:
I'd move this comment above the if statement:

````JavaScript

    // If a URL was passed, the URL of the loaded state is overridden.
    // Note that state is synchronised by the Tuoris service, rather than being broadcast directly to the viewers.
    let url = OVE.Utils.getQueryParam('url');
    if (url) {
        log.debug('New URL at controller:', url);
        window.ove.state.current.url = url;

        log.debug('Caching state');
        window.ove.state.cache();
    } else {
        url = data.url;
    }

````